components/ProfilePage/RepliedArticleTab.js

```diff
@@ -159,6 +159,7 @@ function urlQuery2Filter(query = {}) {
   const filterObj = {
     // Default filters
     replyCount: { GTE: 1 },
+    articleTypes: ['IMAGE', 'TEXT'],
```

Comment:
I think instead of changing individual requests, we can
- use env variable in `rumors-api` to determine if APIs should return all `articleTypes`
    - staging API shows all `articleTypes`; production API only returns `TEXT` by default before we finish implementation
- `rumors-site` does not specify `articleTypes`